rhel/SPECS/pmm-managed.spec

```diff
@@ -12,6 +12,8 @@
 %define release         3
 %define rpm_release     %{release}.%{build_timestamp}.%{shortcommit}%{?dist}
 
+%define full_pmm_version 2.0.0
```

Comment:
Can we break the build if `full_pmm_version` is not defined by the caller? That way we update version only in VERSION file, not in several places like this one.

Reply:
> Can we break the build if `full_pmm_version` is not defined by the caller? That way we update version only in VERSION file, not in several places like this one.

t is also just template, we will change it using the value from vars file

Reply:
I understand that. What I'm saying that it will work incorrectly if we change caller to not overwrite this variable by mistake. Instead, I propose to break the build to make the problem visible. Just like the check I added to `vars` file.

Reply:
> I understand that. What I'm saying that it will work incorrectly if we change caller to not overwrite this variable by mistake. Instead, I propose to break the build to make the problem visible. Just like the check I added to `vars` file.

I got it, let's do it in https://github.com/Percona-Lab/pmm-submodules/pull/271/files#diff-00978888846529056a140f635e7c37d7R51 
if [ -z "${full_pmm_version}" ]; then
    echo 'A  full_pmm_version is not specified.'
    exit 1
fi

Is it ok?

Reply:
OK for me